middleware/gateway/source/manager/admin/cli.cpp

```diff
@@ -120,6 +120,7 @@ namespace casual
                      {
                         using Enum = decltype( value.bound);
                         case Enum::in: return "in";
+                        case Enum::in_forward: return "in*";
```

Comment:
How is the meaning of "in*" communicated to the user?

Reply:
I was looking for a `--legend` to add this info... There seems to be very little documentation about --list-connections, and I thought that was a separate issue.